packages/ui/src/components/BenefitCard/BenefitCard.tsx

```diff
@@ -11,16 +12,23 @@ export interface BenefitCardProps extends CenterProps {
 
 export const BenefitCard = ({ icon, title, subtitle, ...props }: BenefitCardProps) => {
   const [hovered, setHovered] = useState(false);
+  const titleSize = useBreakpointValue({ base: 'md', md: 'lg' } as const);
+  const subtitleSize = useBreakpointValue({ base: 'sm', md: 'md' } as const);
+  const iconWrapperSize = useBreakpointValue({ base: '48px', md: '56px' } as const);
+  const iconSize = useBreakpointValue({ base: '24px', md: '32px' } as const);
+  const wrapperPaddingTop = useBreakpointValue({ base: '44px', md: '52px' } as const);
+  const wrapperPaddingBottom = useBreakpointValue({ base: '40px', md: '80px' } as const);
```

Comment:
💭 All those properties support responsive values by default so there is no need to use `useBreakpointValue` for them